src/components/CustomerQuota/NoQuota/TransactionTitle.tsx

```diff
@@ -13,21 +13,23 @@ export const DistantTransactionTitle: FunctionComponent<{
   toggleTimeSensitiveTitle: boolean;
 }> = ({ transactionTime, toggleTimeSensitiveTitle }) => {
   const { c13nt, i18nt } = useTranslate();
-  const limitReachedDateKey = "limitReachedDate";
+
   const today = toggleTimeSensitiveTitle
     ? `${i18nt("checkoutSuccessScreen", "today")}`
     : "";
   return (
     <>
       <AppText style={sharedStyles.statusTitle}>
-        {c13nt(limitReachedDateKey) !== limitReachedDateKey
-          ? c13nt(limitReachedDateKey)
-              .replace("${dateTime}", formatDateTime(transactionTime))
-              .replace("${today}", today)
-          : i18nt("checkoutSuccessScreen", limitReachedDateKey, undefined, {
-              dateTime: formatDateTime(transactionTime),
-              today,
-            })}
+        {`${c13nt(
+          "limitReachedDate",
+          undefined,
+          i18nt("checkoutSuccessScreen", "limitReachedDate", undefined, {
+            dateTime: formatDateTime(transactionTime),
+            today,
+          })
+        )}`
+          .replace("%{dateTime}", formatDateTime(transactionTime))
+          .replace("%{today}", today)}
```

Comment:
Hmm maybe can add a TODO here to move this functionality into c13nt function, so in the future it offers this parameter feature too~ Same for line 58

Reply:
good call, added [here](https://github.com/rationally-app/mobile-application/pull/508/commits/af4ff260f27365aa0f6dcde211f66beeefa9e360)!